benefits/core/admin/users.py

```diff
@@ -90,16 +89,7 @@ def add_staff_user_to_group(user, request):
         staff_group.user_set.add(user)
 
 
-def add_transit_agency_staff_user_to_group(user, request):
-    user_sso_domain = user.email.split("@")[1]
-    if user_sso_domain:
-        agency = models.TransitAgency.objects.filter(sso_domain=user_sso_domain).first()
-        if agency is not None and agency.staff_group:
-            agency.staff_group.user_set.add(user)
-
-
 def pre_login_user(user, request):
     logger.debug(f"Running pre-login callback for user: {user.username}")
     add_google_sso_userinfo(user, request)
     add_staff_user_to_group(user, request)
-    add_transit_agency_staff_user_to_group(user, request)
```

Comment:
I think this is fine to remove for now. But ideally in the future, when we have SSO enabled for agencies, we would add their users to the "Customer Service" group automatically to avoid manual work from the Cal-ITP side.

We know this helper didn't work during Microsoft SSO sign-in so it would have to be adjusted regardless.